<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>remoteItem</class>
 <widget class="QWidget" name="remoteItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 126, 90)</string>
  </property>
  <widget class="QPushButton" name="sendMsg">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>12</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(255,126,90);
border-radius:13px 13px 13px 13px;</string>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>12</y>
     <width>75</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:rgb(255,126,90);
border-radius:13px 13px 13px 13px;</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>137</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="sysIcon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="sysIcon.qrc">:/sysIcon/icons/redhat.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="deviceName">
        <property name="font">
         <font>
          <family>思源黑体 CN Heavy</family>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>localhost</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="deviceIPv4">
        <property name="font">
         <font>
          <family>思源黑体 CN Heavy</family>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 255, 255)</string>
        </property>
        <property name="text">
         <string>192.168.1.1</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="sysIcon.qrc"/>
 </resources>
 <connections/>
</ui>
